SwiftLeeds/Views/My Conference/MyConferenceViewModel.swift

```diff
@@ -21,6 +21,9 @@ class MyConferenceViewModel: ObservableObject {
             let schedule = try await network.performRequest(endpoint: ScheduleEndpoint())
             self.event = schedule.data.event
             self.slots = schedule.data.slots
+            if let data = try? PropertyListEncoder().encode(slots) {
```

Comment:
This would be better in a Do block to print out any errors for easier debugging.